### PR TITLE
[NMS] Add policy JSON editor

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Add policy JSON editor

### CODE REVIEW

File: nms/app/fbcnms-projects/magmalte/app/views/traffic/TrafficOverview.js [hunk 1]
```diff
@@ -13,22 +13,34 @@
...
  */
+import type {policy_rule} from '@fbcnms/magma-api';
+
 import ApnOverview from './ApnOverview';
 import AppBar from '@material-ui/core/AppBar';
 import Grid from '@material-ui/core/Grid';
 import LibraryBooksIcon from '@material-ui/icons/LibraryBooks';
 import NestedRouteLink from '@fbcnms/ui/components/NestedRouteLink';
 import PolicyOverview from './PolicyOverview';
+
```
Comment (karthiksubraveti) [inline nms/app/fbcnms-projects/magmalte/app/views/traffic/TrafficOverview.js]: minor nit: remove extra line

File: nms/app/fbcnms-projects/magmalte/app/views/traffic/TrafficOverview.js [hunk 2]
```diff
@@ -13,22 +13,34 @@
...
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
 import React from 'react';
 import RssFeedIcon from '@material-ui/icons/RssFeed';
 import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import Text from '@fbcnms/ui/components/design-system/Text';
 
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+
```
Comment (karthiksubraveti) [inline nms/app/fbcnms-projects/magmalte/app/views/traffic/TrafficOverview.js]: same as above

**Review by karthiksubraveti — APPROVED** [review]
Most changes LGTM. Please fix the extra lines and merge.

**Comment by 119Vik** [comment]
ReRunOAITests

**Pull request merged**